ingest/expression_files/dense_ingestor.py

```diff
@@ -188,15 +194,27 @@ def filter_expression_scores(scores: List, cells: List):
                         valid_expression_scores.append(expression_score)
                         associated_cells.append(cells[idx])
             except ValueError as err:
-                raise ValueError(f"Expected numeric expression score - {err}")
+                msg = f"Expected numeric expression score - {err}"
+                GeneExpression.log_for_mixpanel(
+                    "error", "content:type:not-numeric", msg
+                )
+                raise ValueError(msg)
             except Exception:
+                msg = "Score '{expression_score}' is not valid"
+                GeneExpression.log_for_mixpanel(
+                    "error", "content:type:not-numeric", msg
+                )
                 raise ValueError("Score '{expression_score}' is not valid")
         return valid_expression_scores, associated_cells
 
     @staticmethod
     def check_unique_header(header: List):
         """Confirms header has no duplicate values"""
         if len(set(header)) != len(header):
+            msg = "Duplicate header values are not allowed"
```

Comment:
Possible separate task: tell the users [which values are duplicates](https://stackoverflow.com/a/9835819) (perhaps up to 10 duplicates).

Reply:
Good point. More helpful to provide some examples of the duplication. I've added three example duplicates to the error message.
`d18014a`